android/KMAPro/build.sh

```diff
@@ -48,6 +73,20 @@ else
   DAEMON_FLAG=
 fi
 
+# Download default keyboard and dictionary
+if [ $DO_DOWNLOAD = true ]; then
+  downloadPackages
+fi
+
+# Verify default keyboard and dictionary exist
+if [[ ! -f "$KEYBOARDS_TARGET" ]]; then
+  die "$KEYBOARDS_TARGET doesn't exist. Build with -download-resources"
+fi
+
+if [[ ! -f "$MODELS_TARGET" ]]; then
+  die "$MODELS_TARGET doesn't exist. Build with -download-resources"
+fi
```

Comment:
Do I need to unset `KEYBOARDS_TARGET` and `MODELS_TARGET` at the end of this to make sure they don't conflict with the FV build?